testing/python/integration.py

```diff
@@ -234,7 +234,7 @@ def mock_basename(path):
             @mock.patch("os.path.abspath")
             @mock.patch("os.path.normpath")
             @mock.patch("os.path.basename", new=mock_basename)
-            def test_someting(normpath, abspath, tmpdir):
+            def test_someting(normpath, abspath):
```

Comment:
Looking at dba2a8bc647c73eb9ecf21de18bf56e45a04f115 this is deliberate as well - to ensure that `mock.patch` and fixtures work together well.

Reply:
> Looking at dba2a8b this is deliberate as well - to ensure that mock.patch and fixtures work together well.

:+1: restored + renamed c9bb4c4